src/ParamConverterManager.php

```diff
@@ -64,9 +64,10 @@ public function apply(ServerRequest $request, string $controller, string $action
             $methodParam = $methodParams[$i];
             $requestParam = $requestParams[$i];
 
-            $methodParamClass = $methodParam->getClass();
-            if (!empty($methodParamClass)) {
-                $requestParams[$i] = $this->convertParam($requestParam, $methodParamClass->getName());
+            if (!empty($methodParam->getClass())) {
+                $requestParams[$i] = $this->convertParam($requestParam, $methodParam->getClass()->getName());
+            } else {
+                $requestParams[$i] = $this->convertParam($requestParam, $methodParam->getType()->getName(), $methodParam->getDefaultValue());
```

Comment:
@Ali1 the result of `$methodParam->getType()` can be null (this happens when no typehint is used). In this case, the `$methodParam->getType()->getName()` will produce an error (see failed tests)

Reply:
👍